src/runtime/config/configuration-qemu.toml.in

```diff
@@ -246,7 +246,7 @@ vhost_user_store_path = "@DEFVHOSTUSERSTOREPATH@"
 # Enabling this will result in the VM having a vIOMMU device
 # This will also add the following options to the kernel's
 # command line: intel_iommu=on,iommu=pt
-#enable_iommu = true
+enable_iommu = true
```

Comment:
There should be a check to assert that `enable_iommu` and `vfio_mode` are set as expected.

Reply:
I'm not exactly sure what you mean by this.  Do you mean give an error if `enable_iommu = false` and `vfio_mode = vfio` at the same time?  The difficulty with this is that `enable_iommu` is qemu specific, whereas `vfio_mode` is generic.

Reply:
Yes, that's what I mean. Just wanted to rule out the possibility of mis-config rather than having to find it out upon container app failures. I think it can be done in `LoadConfiguration`, no?